.github/workflows/build-container.yml

```diff
@@ -204,3 +209,34 @@ jobs:
 
       - name: Clean up environment
         run: make clean
+
+  push-to-registry:
+    if: github.event_name == 'release' && startsWith(github.ref, 'refs/tags/v')
+    needs: pytest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Download container images
+        uses: actions/download-artifact@v3
+        with:
+          name: images
+
+      - name: Load container images
+        run: |
+          docker load < nvmeof.tar
+          docker load < nvmeof-cli.tar
+          docker load < vstart-cluster.tar
```

Comment:
If we're only pushing `nvmeof` and `nvmeof-cli` I guess we can skip this one... But we'll have to provide separate actions for refreshing the Ceph cluster image (as well as the SPDK image), which can be decoupled from the regular Pull Request CI (both spdk and Ceph images are not likely to change frequently, only on demand).

Reply:
The line removed (thanks).
Another request issue will be opened to handle the Ceph and SPDK images.